ui/components/app/edit-gas-fee-popover/edit-gas-item/edit-gas-item.js

```diff
@@ -147,14 +158,20 @@ const EditGasItem = ({ priorityLevel }) => {
           '--'
         )}
       </span>
+      {/* todo: tooltip in case of swap suggested */}
```

Comment:
I'm guessing we shouldn't consider this a blocker to this PR?

Reply:
I already added tooltip, let me remove this

Reply:
Fixed